### ISSUE TITLE
[Accessibility] The Toot tab at the bottom of the main screen cannot be activated with VoiceOver, regression in beta build 29

### ISSUE THREAD

**Issue by MarcoZehe** [DagAgren/toot#126]
The Toot button (or tab) in the bottom navigation bar of the main screen cannot be activated with VoiceOver. This is a regression in beta build 29. In the first beta from last week, this one worked.

To reproduce:
1. Turn on VoiceOver.
2. Touch the Toot button at the bottom of the main screen.
3. Double-tap to activate.

Result: Nothing happens.
Expected: New Toot screen should open.

Reason: Either the AccessibilityFrame is wrong, or the AccessibilityActivationPoint is somewhere outside the actual control because some relation is wrong. I am blind and cannot tell which is the case, but for a sighted person it should be easily spottable what the problem is.

**Comment by DagAgren** [comment]
That is weird, nothing at all has changed with any of that in this build.

**Comment by MarcoZehe** [comment]
I demonstrated it in the video I direct-messaged you. It‘s 100% reproducible.
